.github/workflows/check.yml

```diff
@@ -64,9 +64,7 @@ jobs:
 
     - name: Install packages
       run: |
-        sudo apt install -y nasm gcc g++ gcc-multilib g++-multilib make binutils grub-pc-bin qemu-system-x86 clang-tidy
-        sudo apt install -y build-essential bison flex libgmp3-dev libmpc-dev
-        sudo apt install -y xorriso mtools inkscape libmpfr-dev texinfo libfuse-dev
+        sudo apt install -y gcc-multilib clang-tidy
```

Comment:
Do you really need `gcc-multilib` ??

Reply:
Yes, for some reason in the CI clang-tidy does not find a header related to bitwise operations, and the only way to fix it was to install `gcc-multilib`.

Reply:
weird :thinking: